__tests__/no-unresolved-refs-error/openapi.yaml

```diff
@@ -0,0 +1,31 @@
+openapi: 3.0.2
+info:
+  title: Example OpenAPI 3 definition.
+  version: 1.0
+
+paths:
+  /pet/findByStatus:
+    get:
+      summary: Finds pets by Status
+      operationId: getPetsByStatus
+      tags:
+        - pets
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ApiResponse'
```

Comment:
I think the idea of __tests__/no-unresolved-refs-error/ test is to check if the error will be returned in case ref is not resolved. So I think we should edit 'no-unresolved-refs-error' test case to not resolve ref and check an error.